From 6c0733b125434b846bec2a744a812c890c82518d Mon Sep 17 00:00:00 2001
From: Takura Mapani <takura.mapani@morsemicro.com>
Date: Tue, 19 Sep 2023 03:25:45 +0000
Subject: [PATCH] Merged in APP-1819_add_usable_by_bannf_column (pull request
 #17)

APP-1819 add a new column usable bannfC for the csv output

* added new column usable bannfC for the csv output


Approved-by: Arien Judge
Approved-by: David Goodall
Approved-by: Matthew Forgie
Approved-by: Thomas Flynn
Approved-by: Matt Jervis
---
 regdbconverter.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/regdbconverter.py b/regdbconverter.py
index 883f281..4bd44b1 100755
--- a/regdbconverter.py
+++ b/regdbconverter.py
@@ -311,7 +311,7 @@ def _output_channels_python(channels, out_file, args):
 
 def _output_channels_csv(channels, out_file, args):
     fields = ['country_code', 'bw', 's1g_chan', 's1g_op_class', 'global_op_class', 'centre_freq_mhz', 'duty_cycle_ap', 'duty_cycle_sta',
-              'country', 'tx_power_max', 'duty_cycle_omit_ctrl_resp', 'pkt_spacing_ms', 'airtime_min_ms', 'airtime_max_ms']
+              'country', 'tx_power_max', 'duty_cycle_omit_ctrl_resp', 'pkt_spacing_ms', 'airtime_min_ms', 'airtime_max_ms', 'usable_banff_c']
 
     dw = DictWriter(out_file, fields, extrasaction='ignore', lineterminator='\n')
     dw.writeheader()
-- 
2.25.1

